packages/runtime/test/transport/relationships.test.ts

```diff
@@ -1016,7 +1017,7 @@ describe("RelationshipDecomposition", () => {
             content: {
                 value: {
                     "@type": "GivenName",
-                    value: "Own name"
+                    value: randomUUID()
```

Comment:
that's not how we create random strings. Also please put this to a variable called ownName to not break the wording here completely.

Also the value below was the same before, now you use a different one for both?

Reply:
Yea the main problem was that they were the same because that's what was disabled and as this function was called multiple times so we needed random values here